src/Tools/Md2Html/Md2Html.csproj

```diff
@@ -30,32 +31,12 @@
     <PackageReference Include="Markdig">
       <Version>0.22.0</Version>
     </PackageReference>
-    <PackageReference Include="NDesk.Options">
-      <Version>0.2.1</Version>
-    </PackageReference>
   </ItemGroup>
   <Target Name="AfterBuildOps" AfterTargets="Build">
-    <Exec Condition=" '$([MSBuild]::IsOSPlatform(`Windows`))' == 'True'"
-		Command="if not exist $(SharedOutputPath) mkdir $(SharedOutputPath)"/>
+    <Exec Condition=" '$([MSBuild]::IsOSPlatform(`Windows`))' == 'True'" Command="if not exist $(SharedOutputPath) mkdir $(SharedOutputPath)" />
     <!-- the ILMergePath property points to the location of ILMerge.exe console application -->
-    <Exec Condition=" '$([MSBuild]::IsOSPlatform(`Windows`))' == 'True'"
-		Command="$(ILMergeConsolePath) $(OutputPath)Md2Html.exe ^
-		/allowDup ^
-		/lib:$(OutputPath) ^
-		/out:$(SharedOutputPath)Md2Html.exe ^
-		AngleSharp.Css.dll ^
-		AngleSharp.dll ^
-		HtmlSanitizer.dll ^
-		Markdig.dll ^
-		NDesk.Options.dll ^
-		System.Buffers.dll ^
-		System.Memory.dll ^
-		System.Numerics.Vectors.dll ^
-		System.Runtime.CompilerServices.Unsafe.dll ^
-		System.Text.Encoding.CodePages.dll " />
-    <Exec Condition=" '!$([MSBuild]::IsOSPlatform(`Windows`))' == 'True' "
-		Command="mkdir -p $(SharedOutputPath)"/>
-    <Exec Condition=" '!$([MSBuild]::IsOSPlatform(`Windows`))' == 'True' "
-		Command="cp $(OutputPath)*.* $(SharedOutputPath)"/>
+    <Exec Condition=" '$([MSBuild]::IsOSPlatform(`Windows`))' == 'True'" Command="$(ILMergeConsolePath) $(OutputPath)Md2Html.exe ^&#xD;&#xA;		/allowDup ^&#xD;&#xA;		/lib:$(OutputPath) ^&#xD;&#xA;		/out:$(SharedOutputPath)Md2Html.exe ^&#xD;&#xA;		AngleSharp.Css.dll ^&#xD;&#xA;		AngleSharp.dll ^&#xD;&#xA;		HtmlSanitizer.dll ^&#xD;&#xA;		Markdig.dll ^&#xD;&#xA;		System.Buffers.dll ^&#xD;&#xA;		System.Memory.dll ^&#xD;&#xA;		System.Numerics.Vectors.dll ^&#xD;&#xA;		System.Runtime.CompilerServices.Unsafe.dll ^&#xD;&#xA;		System.Text.Encoding.CodePages.dll " />
+    <Exec Condition=" '!$([MSBuild]::IsOSPlatform(`Windows`))' == 'True' " Command="mkdir -p $(SharedOutputPath)" />
+    <Exec Condition=" '!$([MSBuild]::IsOSPlatform(`Windows`))' == 'True' " Command="cp $(OutputPath)*.* $(SharedOutputPath)" />
```

Comment:
Do you need to include the commandline parser dll's here in this list?

Reply:
I'm not so sure, but if you consider I can include it.

Reply:
YEs, all references (dlls) should be pacaged up by ILMerge.
You can test it if it works. Not sure if we have tests for Md2Html.exe
@mjkkirschner  ?

Reply:
Solved.

Reply:
m2dhtml.exe should be tested as part of the documentation browser tests, though I don't know if ILMerge works with net6 builds - I have created a task to investigate it.

Reply:
https://jira.autodesk.com/browse/DYN-6021